frontend/src/styles/Courses.css

```diff
@@ -42,8 +42,8 @@
 }
 
 .search-card.both {
-  background-color: #fbf2e9;
-  color: #753a00;
+  background: linear-gradient(to right, #f6ebff, #ebf6f9);
```

Comment:
This looks a bit odd, but I guess I could get used to it.
It will probably make the color scheme more intuitive, so I think it's an improvement.

Reply:
I think it's an improvement over before as well, but I mainly just needed to change the color to something else in order to accommodate external credits. I'll keep thinking about ways to improve our color scheme (for example, now external credits stand out in the schedule more than I'd like them to).